app/models/experimentation/notifications_experiment.rb

```diff
@@ -84,13 +84,13 @@ def self.filter_blocks(patients, blocks)
       end
     end
```

Comment:
You will have to update validate_filters in experiment.rb code to use facility_slugs instead of facilities?

Reply:
Oops, thanks missed checking that. It doesn't seem necessary with the current naming convention (since they all use different fields). But it will be clear what is expected. Shall I also change states and blocks to `state_names` and `block_ids`?